src/components/DefineDissolution/CustodianOfRecords.vue

```diff
@@ -203,7 +203,9 @@
           </v-col>
           <v-col cols="12" md="5" lg="5" class="pl-8">
             <label class="summary-sub-label">Delivery Address</label>
-            <div v-if="isSame(getCustodian.mailingAddress, getCustodian.deliveryAddress)">
+            <div v-if="getCustodian.inheritMailingAddress &&
+              isSame(getCustodian.mailingAddress, getCustodian.deliveryAddress)"
```

Comment:
How could one of these conditions be true but not the other?

How does a user change from a different address to the same address?

Reply:
Good point, if it's selected they will always be the same. The second condition is redundant. Will remove 👍